addons/hr/views/res_users.xml

```diff
@@ -137,70 +67,17 @@
-                                <div class="o_row">
-                                    <field name="distance_home_work" readonly="not can_edit" class="o_hr_narrow_field"/>
-                                    <span><field name="distance_home_work_unit" readonly="not can_edit"/></span>
-                                </div>
-                            </group>
-                            <group string="Citizenship">
-                                <field name="employee_country_id" options='{"no_open": True, "no_create": True}' readonly="not can_edit"/>
-                                <field name="identification_id" readonly="not can_edit"/>
-                                <field name="ssnid" readonly="not can_edit"/>
-                                <field name="passport_id" readonly="not can_edit"/>
-                                <field name="sex" readonly="not can_edit"/>
-                                <label for="birthday"/>
-                                        <div class="oe_inline">
-                                            <field name="birthday" class="o_hr_narrow_field" readonly="not can_edit"/>
-                                            <span invisible="not birthday" groups="hr.group_hr_user">
-                                                <label for="birthday_public_display" class="fw-bold text-900 form-check-label ms-3"/>
-                                                <field name="birthday_public_display" class="ms-3" readonly="not can_edit"/>
-                                            </span>
-                                        </div>
-                                <field name="place_of_birth" readonly="not can_edit"/>
-                                <field name="country_of_birth" readonly="not can_edit"/>
-                            </group>
-                            <group string="Marital Status">
-                                <field name="marital" readonly="not can_edit"/>
-                                <field name="spouse_complete_name" invisible="marital not in ['married', 'cohabitant']" readonly="not can_edit"/>
-                                <field name="spouse_birthdate" invisible="marital not in ['married', 'cohabitant']" readonly="not can_edit"/>
-                            </group>
-                            <group string="Education">
-                                <field name="certificate" readonly="not can_edit"/>
-                                <field name="study_field" readonly="not can_edit"/>
-                                <field name="study_school" readonly="not can_edit"/>
+                                <field name="private_email" string="Private Email" readonly="not can_edit"/>
+                                <field name="private_phone" string="Private Phone" class="o_force_ltr" readonly="not can_edit"/>
+                                <field name="bank_account_id" readonly="not is_hr_user" context="{'default_partner_id': work_contact_id}"/>
```

Comment:
do we have access to is_hr_user since we remove it from the view below? :thinking: